base_layer/wallet/tests/support/utils.rs

```diff
@@ -81,12 +82,18 @@ pub fn make_input<R: Rng + CryptoRng>(
     rng: &mut R,
     val: MicroTari,
     factory: &CommitmentFactory,
-) -> (TransactionInput, UnblindedOutput)
+) -> (TransactionInput, DbUnblindedOutput)
```

Comment:
I disagree with this. The DbUnblindedOutput should only be used within the OMS but this utility function is used to test many things in the wallet as a whole so by doing this now all the tests for the other services need to use the .into() everywhere and have to include the DbUnlindedOutput implementation.